src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -564,11 +564,12 @@ private void markAsRead() {
         }
     }
 
-    private void createNewWatchListMenuItem() {
+    /**
+     * Creates a new watch list panel containing the currently selected issue
+     */
+    private void createNewWatchListPanel() {
         Optional<GuiElement> item = listView.getSelectedItem();
-        if (!item.isPresent()) {
-            return;
-        }
+        assert item.isPresent();
```

Comment:
Yes, the chances are very low. Let's go ahead and show an error message saying the operation failed because the issue disappeared from the view before it can be updated.